indexing-hadoop/src/main/java/io/druid/indexer/partitions/SingleDimensionPartitionsSpec.java

```diff
@@ -57,4 +60,15 @@ public Jobby getPartitionJob(HadoopDruidIndexerConfig config)
   {
     return new DeterminePartitionsJob(config);
   }
+
+  @Override
+  @JsonProperty
+  public List<String> getPartitionDimensions()
+  {
+    if (partitionDimension != null) {
+      return ImmutableList.of(partitionDimension);
+    } else {
+      return null;
```

Comment:
minor not: would it make sense and make code more clean to make partitonDimensions an empty list when user does not specify any dimensions ?